core/src/cats/bdb_dbi.h

```diff
@@ -39,18 +39,14 @@ struct DbiFieldGet {
 
 class BareosDbDBI : public BareosDbPrivateInterface {
  private:
-  /*
-   * Members.
-   */
+  // Members.
   dbi_inst instance_;
   dbi_conn* db_handle_;
   dbi_result* result_;
   DbiFieldGet* field_get_;
 
  private:
-  /*
-   * Methods.
-   */
+  // Methods.
```

Comment:
... and its also obvious that these are methods. I guess we all have seen method signatures in the past.